bootstrap-sass/src/main/java/de/agilecoders/wicket/sass/UrlImporter.java

```diff
@@ -231,8 +232,8 @@ private String addUnderscore(String url) {
 
     private String getAbsolutePath(URI base, String url) {
         String basePath = base.toString();
-        Path parentBasePath = Paths.get(basePath).getParent();
-        return parentBasePath.resolve(url).toString();
+        String parentBasePath = basePath.substring(0, basePath.lastIndexOf('/'));
```

Comment:
I don't have Windows to test it but why `lastIndexOf('/')` ? Shouldn't it be `\` for Windows ?

Reply:
This is a URL not a path, so it is also a frontslash on Windows.